smac/utils/sklearn_wrapper.py

```diff
@@ -0,0 +1,324 @@
+'''
+Created on July 26, 2017
+
+@author: Jan N. van Rijn
```

Comment:
Please drop this docstring
and add some module attributes such as
__author__ = "Jan N. van Rijn"
__license__ = "3-clause BSD"